.gitignore

```diff
@@ -13,3 +12,10 @@ src/stamp-h1
 src/*.o
 VERSION
 *.tar.gz
+
+# These auto-generated files will form part of release tarballs but are
+# not needed when building from the repository as the autotools would
+# already need to be installed and can therefore recreate them:
+/build-aux/*
+/src/config.h.in
+/configure
```

Comment:
Also not have a leading slash. And therefore line 21 is the same as the original line 5.